newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -70,6 +74,14 @@ export default function SummaryPage() {
 
   return (
     <Container text>
+      {hasParticipants > 0 ? (
```

Comment:
If you do `{hasParticipants > 0 && (...)}` you can avoid the `: null` at the end